From c128ffb9f901cb14b0d36ee8470d0e18727ac520 Mon Sep 17 00:00:00 2001
From: Andy Grover <agrover@redhat.com>
Date: Tue, 4 Jun 2013 22:34:48 -0700
Subject: [PATCH] fix-exception-in-pscsi-backstore

---
 rtslib/tcm.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rtslib/tcm.py b/rtslib/tcm.py
index 9a8e201..62bd7bb 100644
--- a/rtslib/tcm.py
+++ b/rtslib/tcm.py
@@ -270,7 +270,7 @@ class PSCSIStorageObject(StorageObject):
 
     def _configure(self, dev):
         self._check_self()
-        parent_hostid = self.backstore.index
+        parent_hostid = self._backstore.index
 
         # Use H:C:T:L format or preserve the path given by the user.
         try:
